ui/src/app/certificate_requests/table.tsx

```diff
@@ -0,0 +1,38 @@
+import { Dispatch, SetStateAction, useContext } from "react"
+import { AsideContext } from "../nav"
+import Row from "./row"
+
+
+export function CertificateRequestsTable() {
+    const { isOpen: isAsideOpen, setIsOpen: setAsideIsOpen } = useContext(AsideContext)
+    return (
+        <div className="p-panel">
+            <div className="p-panel__header is-sticky">
+                <h4 className="p-panel__title">Certificate Requests</h4>
+                <div className="p-panel__controls">
+                    <button className="u-no-margin--bottom p-button--positive" aria-label="add-csr-button" onClick={() => setAsideIsOpen(true)}>Add New CSR</button>
+                </div>
+            </div>
+            <div className="p-panel__content">
+                <div className="u-fixed-width">
+                    <table aria-label="Certificate Requests Table" className="p-main-table">
+                        <thead>
+                            <tr>
+                                <th>ID</th>
```

Comment:
It feeld like the ID in the table takes a lot of room for someting that is not really human readable. It feels like the real differentiator for humans would be the subject alternative names but that's a list of elements.

What do you think?

Reply:
I think the ID is important simply because according to the spec, the pebble notices share ID's when a certificate is updated. On top of that, we use the ID to refer to each entry in the row with the API, like `certificate_request/{id}`. I can imagine a scenario where a user may want to know the ID so that they can manage it through the API.

I do agree that it takes a lot of room, I want to shrink it down to the width of just 4 characters, but I haven't found how to yet in the vanilla framework.

Reply:
I'm still undecided if we should keep the ID's.